cmd/adhoc/cmd/root.go

```diff
@@ -30,18 +33,31 @@ func onPersistentPreRun(cmd *cobra.Command, args []string) error {
 		// Set up logging
 		func() error {
 			parser := cli.NewParserForCobra(cmd, args)
-
 			return log.Configure(parser)
 		},
+		func() error {
+			// Setting up file formatting
+			viper.SetConfigFile("syncron.yaml")
+			viper.SetConfigType("yaml")
+			viper.AddConfigPath(".")
+			// Reading from file
+			err := viper.ReadInConfig()
+			if err != nil {
+				return err
+			} else {
+				logrus.Info("Your configuration file was read succesfully!")
+				bucket_name := viper.Get("buckets.sosreports")
+				logrus.Info("Reading from bucket: ", bucket_name)
+			}
+			return err
```

Comment:
Nitpicking here, 'return nil' instead to make it clear that at that point there will be no error.

Reply:
good catch.
fixed!